package.json

```diff
@@ -67,6 +67,7 @@
     "rollup": "^0.41.6",
     "rollup-plugin-filesize": "^1.3.2",
     "rollup-plugin-node-resolve": "^3.0.0",
+    "serializr": "^1.3.0",
```

Comment:
I added `serializr` as a dev dependency just for this [test](https://github.com/mobxjs/mobx/pull/1652/files#diff-2492cdcb925c1d209eec08acc8ff1320R428), so I can compose `@observable` with another decorator `@serializable(primitive())`.

Reply:
N.P.